spec/rubocop/cop/rails/pluralization_grammar_spec.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::PluralizationGrammar do
+  subject(:cop) { described_class.new }
+  before(:each) do
+    inspect_source(cop, source)
+  end
+
+  shared_examples 'enforces pluralization grammar' do |method_name|
+    context "When #{method_name} is called on an unknown variable" do
+      context "when using the plural form ##{method_name}s" do
+        let(:source) { "some_variable.#{method_name}s" }
+        it 'does not register an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+
+      context "when using the singular form ##{method_name}" do
+        let(:source) { "some_method.#{method_name}" }
+
+        it 'registers an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+    end
+
+    [-1, -1.0, 1, 1.0].each do |singular_number|
```

Comment:
Take a look at `TextUtil#pluralize`. Internally, RuboCop says that `-1` should be pluralized.

I am having trouble finding the comments in the PR talking about why we went that way, but I do remember doing a bit of research around what is grammatically correct. I think we found either is acceptable, but plural is preferred.

This seems to generate much debate based on context and logical grouping http://english.stackexchange.com/questions/9735/is-1-singular-or-plural.

From the sound of it, it comes down to how you interpret it `(-1.days)` vs `-(1.day)`